spec/controllers/exercise_solutions_controller_spec.rb

```diff
@@ -55,7 +56,8 @@ def post_problem(problem)
 
       it { expect(response.body).to json_eq({ status: :failed, guide_finished_by_solution: false },
                                             except: [:class_for_progress_list_item, :html, :remaining_attempts_html,
-                                                     :title_html, :button_html, :expectations, :test_results, :tips]) }
+                                                     :title_html, :button_html, :expectations, :test_results, :tips,
+                                                     :current_exp]) }
```

Comment:
Could you please update JS type information and response format documentation in `README` of current_exp?